### DISCUSSION TITLE
EPGS Code for oceanografic 3d data with lon/lat + depth in dbar?

### DISCUSSION THREAD

**Discussion by gannebamm** [INSPIRE-MIF/helpdesk#159] [Q&A]
The Thünen-Institute surveys oceanographic datasets. These datasets contain vertical profiles of temperature and salinity taken at haul positions. The datasets contain vertical temperature and salinity profiles, proceeded according to the Intergovernmental Oceanographic Commission (IOC), which are interpolated vertically into 1-dbar intervals. Therefore, they need a reference system that contains WGS84 lon/lat + metric heights. Our scientists discussed the use of https://epsg.io/4979. This EPSG is not part of the official registry and will not be validated. 

Maybe they should use this one instead: http://www.opengis.net/def/crs/EPSG/0/9390
 • EVRF2019 mean-tide height EPSG:9390

Example data (non-3D): https://atlas.thuenen.de/layers/geonode_data_ingest:geonode:hydro_survey_megs

**Comment by fabiovinci** [comment]
Dear @gannebamm,

if you can manage to change the reference system of the dataset it is ok since the EPSG:9390 was already added to the official list of CRS approved for use in INSPIRE (see [relevant table](https://github.com/INSPIRE-MIF/helpdesk/blob/main/crs.md)).

I will convert the issue into a discussion to leave it open for additional input.

If you will consider that the EPSG:4979 is relevant for these types of datasets, you can open a new issue to ask for its inclusion in the official list.
